en/guide/configuration_file.md

```diff
@@ -109,18 +109,50 @@ Example (Ubuntu):
 video0=http://127.0.0.1:8000/camera-def-uvc.xml
 ```
 
-### [imgcap]
+### [imgcap] {#imgcap}
+
+<!-- These specify the default values for image capture, that will eventually be over ridable by parameter settings from MAVLink -->
 
 Key | Description | Default
 -- | --- | ---
-`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory) | -
+`width` | Width of the image to be captured in pixels. | Full width of camera frame for sensor type (i.e 1080P - 1920, 720P - 1280, etc.)
+`height` | Height of the image to be captured in pixels. | Full height of camera frame for sensor type (i.e 1080P - 1080, 720P - 720, etc).
+`format` | Image format (number). <br>Possible values:<ul><li>2 (Jpeg/<code>IMAGE_FILE_JPEG</code>)</li></ul>Notes:<ul><li>Currently only Jpeg is <a href="https://github.com/Dronecode/camera-streaming-daemon/issues/163">supported for image capture</a>.</li><li>At time of writing this cannot be overwritten in a <a href="../guide/camera_definition_file.md">Camera Definition File</a> (see <a href="https://github.com/Dronecode/camera-streaming-daemon/issues/161">#161</a>).</li></ul> | 2 (JPEG).
+`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory). The path should be accessible and writeable. | /tmp/
 
 Example:
 ```
 [imgcap]
 location=~/Temp/
 ```
 
+### [vidcap] {#vidcap}
+
+<!-- These specify the default values for video capture, that will eventually be over ridable by parameter settings from MAVLink -->
```

Comment:
@lbegani My understanding is that **all** vidcap and imagecap keys all define _default_ values for image/video capture that it should be possible to update via mavlink (ie QGC) once https://github.com/Dronecode/camera-streaming-daemon/issues/161 is done. Is that correct?

If so, I'll add a note about that at the top of these sections.

Reply:
Yes, other than `location`, everything else should be settable

Reply:
Thanks very much. I have updated. This is now ready for review :-)